README.md

```diff
@@ -1 +1,2 @@
-# ot-scanner-libs
\ No newline at end of file
+# ot-scanner-libs
+I am placeholder of all the python libraries related to Images and Credentials Scanning
```

Comment:
READEME should be proper ...  Mention what inputs and type of inputs this lib required